tests/test_ctl.py

```diff
@@ -402,9 +402,15 @@ def test_get_all_members(self):
     @patch('patroni.ctl.get_dcs')
     def test_members(self, mock_get_dcs):
         mock_get_dcs.return_value.get_cluster = get_cluster_initialized_with_leader
+
         result = self.runner.invoke(ctl, ['list'])
         assert '127.0.0.1' in result.output
         assert result.exit_code == 0
+        assert 'Citus cluster: alpha (12345678901)' in result.output
```

Comment:
It shows a system id for coordinator cluster if patronictl is executed without `--group` argument.
This is not the behavior we want, because users might get a false impression that all nodes in the citus cluster have the same sysid, while in fact it is different for different groups.

Reply:
🤦‍♀️ sure